app/models/cloud9_invite.rb

```diff
@@ -0,0 +1,16 @@
+class Cloud9Invite < ApplicationRecord
+  validates :email, presence: true
+  validates :email, uniqueness: true
+  validates :email, email: true
+
+  before_create :invite_member
+
+  def invite_member
+    Cloud9Client::Team.invite_member(
+      Rails.application.secrets.cloud9_team_name,
+      email
+    )
+  rescue
+    errors.add(:invite_member_error, 'Couldn\'t invite member')
+  end
```

Comment:
One last thing: if you choose to specifically rescue from a 409 conflict response, you want to make sure you return the exact same message as a uniqueness validation conflict. Make sure to write a test for this case.